src/analysis/calculus/cont_diff.lean

```diff
@@ -1327,6 +1327,12 @@ lemma cont_diff_at_zero :
   cont_diff_at 𝕜 0 f x ↔ ∃ u ∈ 𝓝 x, continuous_on f u :=
 by { rw ← cont_diff_within_at_univ, simp [cont_diff_within_at_zero, nhds_within_univ] }
 
+theorem cont_diff_at_one_iff : cont_diff_at 𝕜 1 f x
+  ↔ ∃ f' : E → (E →L[𝕜] F), ∃ u ∈ 𝓝 x, continuous_on f' u ∧ ∀ x ∈ u, has_fderiv_at f (f' x) x :=
```

Comment:
I expected the `\iff` at the end of the line above. What is the convention?

Reply:
I agree